modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3BlobStore.java

```diff
@@ -172,6 +177,7 @@ public final void collectMetrics(Request<?> request, Response<?> response) {
                 assert assertConsistencyBetweenHttpRequestAndOperation(request, operation);
                 counter.add(getRequestCount(request));
                 requestCounter.incrementBy(getRequestCount(request), attributes);
+                httpRequestTimeInMicrosCounter.incrementBy(getHttpRequestTimeInMicros(request), attributes);
```

Comment:
Similar to the APM metric above, this should multiplex to the various use-cases, as per the `attributes` map.